src/meta/timing.rs

```diff
@@ -145,51 +159,63 @@ impl TimedCode {
 
     /// Start the next measurement
     pub fn start(&mut self) {
-        if cfg!(test) {
+        if cfg!(test) || cfg!(not(feature = "timing")) {
```

Comment:
This could be a compile-time check instead of a runtime check. Maybe it is nicer to provide a second implementation?
```rust
    #[cfg(any(feature = "test", not("feature = timing")))]
    pub fn start(&mut self) {
         // No-Op 
    }
```